src/api/publicationChannelApi.ts

```diff
@@ -0,0 +1,28 @@
+import Axios from 'axios';
+
+import { PUBLICATION_CHANNEL_API_URL, PublicationTableNumber } from '../utils/constants';
+
+interface PublicationChannelItem {
+  originalTitle: string;
+  printIssn: string;
+  onlineIssn: string;
+  level: number | null;
+  publishing: string;
+}
```

Comment:
i develop har vi atm to typer: `Publication` og `PublicationChannel`, som jeg i min PR har slått sammen til en type:
```
export interface Publisher {
  title: string;
  issn: string;
  level: string;
  publisher: string;
}
```
Jeg har ikke helt kontroll på hva som trengs av disse, men virker som om det burde holde med en type(?)

Reply:
den typen over der er det vi får fra backend. vi bør kanskje skille på de. men tenker vi kan ta diskusjonen etter vi har sett på din PR